metadata-ingestion/tests/conftest.py

```diff
@@ -49,3 +51,40 @@ def pytest_addoption(parser):
         default=False,
     )
     parser.addoption("--copy-output-files", action="store_true", default=False)
+
+
+def pytest_collection_modifyitems(
+    config: pytest.Config, items: List[pytest.Item]
+) -> None:
+    # https://docs.pytest.org/en/latest/reference/reference.html#pytest.hookspec.pytest_collection_modifyitems
+    # Adapted from https://stackoverflow.com/a/57046943/5004662.
+
+    root = pathlib.Path(config.rootpath)
+    integration_path = root / "tests/integration"
+
+    for item in items:
+        test_path = pathlib.Path(item.fspath)
+
+        if (
+            "docker_compose_runner" in item.fixturenames  # type: ignore[attr-defined]
+            or any(
+                marker.name == "integration_batch_2" for marker in item.iter_markers()
+            )
+        ):
+            item.add_marker(pytest.mark.slow)
```

Comment:
Can we manually mark each test in this batch slow instead? Feels like this is something that'd be easy to miss

Reply:
we don't actually use `slow` anywhere, so I think it's fine - I mainly added it as a convenience thing to run quick unit tests locally